frontend/packages/schema-editor/src/components/SchemaInspector/restrictions/NumberRestrictionsReducer.tsx

```diff
@@ -1,11 +1,12 @@
+import { NameError } from '@altinn/schema-editor/types';
```

Comment:
Du kan godt lage et eget enum for disse feilmeldingene. `NameError` er laget for en spesifikk type feil som oppstår et annet sted.

Reply:
Fikset